airflow/dags/libs/news_article/db_update/article_final_selection.py

```
@@ -85,68 +78,54 @@ def insert_content(article_id, article_content):
+        mongo_db = get_mongo_connection()
+        selected_articles = mongo_db["Selected_Articles"]
 
-        # 2. JSON 문자열 변환
-        categories = json.dumps(category_names, ensure_ascii=False)
-        
-        # 필수 필드 검증
-        required_fields = ["url", "title","sub_col", "content_col", "author", "publish_time"]
+        required_fields = ["url", "title", "sub_col", "content_col", "author", "publish_time"]
         for field in required_fields:
             if not article_content.get(field):
                 raise ValueError(f"[article_id={article_id}] 필수 필드 누락: {field}")
```

Ruff (0.14.3)

Abstract `raise` to an inner function

(TRY301)